Makefile

```diff
@@ -308,7 +308,7 @@ runworker:
 	@if [ "$(VENV_BASE)" ]; then \
 		. $(VENV_BASE)/awx/bin/activate; \
 	fi; \
-	$(PYTHON) manage.py runworker --only-channels websocket.*
+	$(PYTHON) manage.py runworker --only-channels websocket.* --only-channels persistence
```

Comment:
Again this name dosen't convey anything useful. Can we call it something like `/networking_events`

Reply:
I changed Persistence to NetworkingEvents and removed the extra channel completely.

Reply:
I removed the try/except since that was used for development to generate a console log statement on server-side errors happened.  There was no corresponding UI error notification.